app/models/concerns/image_file.rb

```diff
@@ -106,7 +106,12 @@ def reset_image_attributes
   end
 
   def detect_image_attributes
-    return if !original_url || (width && height && format)
+    # skip if we've already detected width/height/format
+    return if width.present? && height.present? && format.present?
+
+    # s3 urls cannot be fastimage'd - must wait for async porter inspection
+    return if original_url.blank? || original_url.starts_with?("s3://")
```

Comment:
Still some TODOs in #618 for how we handle async validation of these `s3://` uploads from Feeder.  But this at least makes it work ... don't try to use FastImage to detect image height/width synchronously, if the url starts with s3.